mime-ballerina/byte_stream.bal

```diff
@@ -42,16 +45,22 @@ class ByteStream {
     }
 
     # The next function reads and return the next `byte[]` of the related stream.
+    # ```ballerina
+    # record {|byte[] value;|}|io:Error? data = streamer.next();
```

Comment:
Since this is an anti-pattern, let's make use of `check`.

Reply:
Do we need it for the function API doc also? I feel like it hide some details if we use check. For example, here the returned error is `io:Error` and if we hide it using a `check` docs are not informative. This check is suitable for a code segment IMO. This doc is just to explain more on the function structure right? WDYT?

Reply:
Yeah, I also felt that function API doc might not be the best place to use`check`. However, as I remember it was agreed in the meeting to use it in function API docs as well since it is the best practice, and some modules have already changed. @shafreenAnfar , @daneshk WDYT.